ec2-dash/dash_sqlite.py

```diff
@@ -185,18 +190,30 @@ def push_to_s3():
                 most_recent_ride_to_csv(cursor)
                 user_info_to_csv(cursor)
                 push_to_s3()
-                create_new_current_ride_table(cursor, conn)
-                os.remove("ec2-dash/most_recent_ride.csv") # delete the old log csv
-                try:
-                    os.remove("ec2-dash/user_info.csv") # delete the old user csv
-                    create_user_info_table(cursor, conn, log_entry)
-                except:
-                    create_user_info_table(cursor, conn, log_entry)
+                
+                # delete expired csv files 
+                os.remove("ec2-dash/most_recent_ride.csv") 
+                os.remove("ec2-dash/user_info.csv") 
+                os.remove("ec2-dash/join_ids.csv") 
+
+                #regenerate the tables
+                recreate_current_ride_table(cursor, conn)
+                recreate_user_info_table(cursor, conn, log_entry)
+
+                # regenerate the join_ids and user_info csvs
+                ride_id = recreate_ride_id_from_datetime(log_entry)
+                ride_date = log_entry['date'] + " " + log_entry['time']
+                recreate_join_csv(log_entry['user_id'], ride_id)
+
```

Comment:
For the ride csv, we are including the user id and ride id along with the aggregated data for each ride as one row, so the separate join table is no longer needed

Reply:
Also done. Only two csv files are now created and pushed to s3, the user info and the log info for the most recent ride